sidebars.js

```diff
@@ -158,12 +158,14 @@ const sidebars = {
   'spl-docs/search-command',
   'spl-docs/sort-command',
   'spl-docs/stats-command',
+  'spl-docs/streamstats-command',
   'spl-docs/tail-command',
   'spl-docs/time-modifiers',
   'spl-docs/timechart-command',
   'spl-docs/top-command',
   'spl-docs/transaction-command',
   'spl-docs/where-command',
+  'spl-docs/gentimes-command',
```

Comment:
nit: this should be placed alphabetically

Reply:
addressed in the latest commit.